python/tests/test_logistic_regression.py

```diff
@@ -291,8 +304,13 @@ def test_classifier(
     reg_param: float = 0.0,
     elasticNet_param: float = 0.0,
     tolerance: float = 0.001,
+    standardization: bool = False,
     convert_to_sparse: bool = False,
 ) -> LogisticRegression:
+    assert (
+        standardization is False
+    ), "standardization=True is not supported due to testing with single-GPU LogisticRegression"
```

Comment:
Can we use cuml standard scalar preprocessr?

Reply:
revised the code to remove the standardization option from argument list. 
this test was for standardization=False only.